packages/hierarchies-react/src/presentation-hierarchies-react/Tree.tsx

```diff
@@ -50,6 +50,7 @@ export function TreeRenderer({
                 className="filtering-action-button"
                 styleType="borderless"
                 size="small"
+                title="Clear active filter"
```

Comment:
need to think about our localization strategy in this package

Reply:
filed a separate issue for tracking localization: https://github.com/iTwin/presentation/issues/490